aleph-client/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aleph_client"
-version = "3.9.0"
+version = "3.9.1"
```

Comment:
Nitpick: since we're changing this line, let's bump to `3.10.0`. I know it does not matter since we don't publish it anywhere but for sake being compliant with semver we should bump minor version, as we're adding new methods to API of `aleph-client`.

Reply:
oh, yes, totally true - I'm sorry, a moment earlier I was working with drink, where I'm bumping always patch component (it's still at `0.x.x`) :see_no_evil: